test/index.js

```diff
@@ -626,6 +633,332 @@ describe('onPostBuild', () => {
       },
     ])
   })
+
+  test('adds headers to Netlify configuration', async () => {
+    await moveNextDist()
+
+    const show = jest.fn()
+
+    await plugin.onPostBuild({
+      ...defaultArgs,
+
+      utils: { ...defaultArgs.utils, status: { show }, functions: { list: jest.fn().mockResolvedValue([]) } },
+    })
+
+    expect(netlifyConfig.headers).toEqual([
+      {
+        for: '/',
+        values: {
+          'x-custom-header': 'my custom header value',
+        },
+      },
+      {
+        for: '/en/',
+        values: {
+          'x-custom-header': 'my custom header value',
+        },
+      },
+      {
+        for: '/es/',
+        values: {
+          'x-custom-header': 'my custom header value',
+        },
+      },
+      {
+        for: '/fr/',
+        values: {
+          'x-custom-header': 'my custom header value',
+        },
+      },
+      {
+        for: '/api/*',
+        values: {
+          'x-custom-api-header': 'my custom api header value',
+        },
+      },
+      {
+        for: '/en/api/*',
+        values: {
+          'x-custom-api-header': 'my custom api header value',
+        },
+      },
+      {
+        for: '/es/api/*',
+        values: {
+          'x-custom-api-header': 'my custom api header value',
+        },
+      },
+      {
+        for: '/fr/api/*',
+        values: {
+          'x-custom-api-header': 'my custom api header value',
+        },
+      },
+      {
+        for: '/*',
+        values: {
+          'x-custom-header-for-everything': 'my custom header for everything value',
+        },
+      },
+      {
+        for: '/en/*',
+        values: {
+          'x-custom-header-for-everything': 'my custom header for everything value',
+        },
+      },
+      {
+        for: '/es/*',
+        values: {
+          'x-custom-header-for-everything': 'my custom header for everything value',
+        },
+      },
+      {
+        for: '/fr/*',
+        values: {
+          'x-custom-header-for-everything': 'my custom header for everything value',
+        },
+      },
+    ])
+  })
+
+  test('appends headers to existing headers in the Netlify configuration', async () => {
```

Comment:
[boulder] this test and it's contents appear twice in this file

Reply:
Oof. How'd that get in there? 😬